src/api/window/window.ts

```diff
@@ -887,6 +887,15 @@ export class _Window extends Base {
         })).then(() => undefined);
     }
 
+    /**
+     * Shows the Chromium Developer Tools
+     * @return {Promise.<void>}
+     * @tutorial Window.showDeveloperTools
+     */
+    public showDeveloperTools(): Promise<void> {
+        return this.wire.sendAction('show-developer-tools', this.identity).then(() => undefined);
+    }
+
```

Comment:
This is System method, not window.

Reply:
I know, want to move to Window as it is more natural. 

`win.showDeveloperTools();`
versus 
`fin.System.showDeveloperTools('uuid', 'name');`

Reply:
i see. That means core needs to update too, right?

Reply:
The core would need no changes.